source/Octopus.Manager.Tentacle.Tests/packages.config

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file doesn't exist in the other test projects either, does it need to exist here?